packages/create-atomic/plopfile.js

```diff
@@ -0,0 +1,58 @@
+const {spawn} = require('child_process');
+
+module.exports = function (
+  /** @type {import('plop').NodePlopAPI} */
+  plop
+) {
+  const currentPath = process.cwd();
+  plop.setGenerator('@coveo/atomic', {
+    description: 'A Coveo Atomic Generator',
+    prompts: [
+      {
+        type: 'input',
+        name: 'project',
+        message: 'Name of the project',
+        validate: (input) => {
+          const pattern =
+            '^(?:@[a-z0-9-*~][a-z0-9-*._~]*/)?[a-z0-9-~][a-z0-9-._~]*';
+          if (!new RegExp(pattern).test(input)) {
+            return `The project name should match the pattern "${pattern}" (https://docs.npmjs.com/cli/v7/configuring-npm/package-json#name)`;
```

Comment:
What would be the potential problem by letting users input "anything" for the project name ? 
Do we really need to validate here ?

Because the pattern is pretty convoluted, so I'm not sure, as an end user, that I would be able to figure out "oh yeah I should change this in my project name".

Maybe adding an example of a valid name ?